wdio/features/screen-objects/WalletMainScreen.js

```diff
@@ -3,7 +3,7 @@ import { ONBOARDING_WIZARD_STEP_1_CONTAINER_ID } from '../testIDs/Components/Onb
 class WalletMainScreen {
   async validateOnboardingWizard() {
     const elem = await $(`~${ONBOARDING_WIZARD_STEP_1_CONTAINER_ID}`);
-    expect(elem).toBeDisplayed();
+    await expect(elem).toBeDisplayed();
```

Comment:
Create a getter for the object then assert that the object is visible. Something like:


```suggestion

  get onBoardingWizard() {
    return Selectors.getElementByPlatform(ONBOARDING_WIZARD_STEP_1_CONTAINER_ID);
  }

  async isOnboardingWizardVisible() {
    await expect(this.onBoardingWizard).toBeDisplayed();
  }
```

Reply:
This file was implemented by either Yande or Fatia.  I just made a simple fix by adding await in the return.  I would actually like this file to be renamed to "TakeATourComponent.js" or something like that and probably best to do maintenance work on another PR.

Reply:
That is fair.